runtime/src/comm/gasnet/comm-gasnet-ex.c

```diff
@@ -893,10 +895,13 @@ void chpl_comm_init(int *argc_p, char ***argv_p) {
   setup_ibv();
   setup_polling_pre_init();
 
+  defer_gasnet_progress_threads = chpl_env_rt_get_bool("COMM_GASNET_DEFER_PROGRESS_THREADS", false);
+
   GASNET_Safe(gex_Client_Init(&myclient, &myep, &myteam, "chapel",
                               argc_p, argv_p,
                               GEX_FLAG_USES_GASNET1 |
-                              GEX_FLAG_DEFER_THREADS));
+                              (defer_gasnet_progress_threads ?
+                               GEX_FLAG_DEFER_THREADS : 0)));
```

Comment:
I think the flags should be split up for better readability
```
gex_Flags_t flags = GEX_FLAG_USES_GASNET1 |
                                 (defer_gasnet_progress_threads ? GEX_FLAG_DEFER_THREADS : 0);
```
But this is arguably style preference and I am fine with it as-is

Reply:
Addressed in 1b982b85